mapie/tests/test_regression.py

```diff
@@ -748,3 +748,20 @@ def test_predict_infinite_intervals() -> None:
     _, y_pis = mapie_reg.predict(X, alpha=0., allow_infinite_bounds=True)
     np.testing.assert_allclose(y_pis[:, 0, 0], -np.inf)
     np.testing.assert_allclose(y_pis[:, 1, 0], np.inf)
+
+
+@pytest.mark.parametrize("method", ["minmax", "naive", "plus"])
```

Comment:
You can add the `base` method to check that we preserve its value.
```suggestion
@pytest.mark.parametrize("method", ["minmax", "naive", "plus", "base"])
```

Reply:
Yes indeed, thank you Thibault for your remark !